.gitignore

```diff
@@ -0,0 +1,5 @@
+# compiled output
+/dist
+
+# dependencies
+/node_modules
```

Comment:
Here you can see the message from GitHub 'No newline at end of file'
You can configure your editor with .editorconfig to automatically add newline on save

Reply:
add newline